commit 508cb342d8f88e9a1d25d6aec78a43b7050b4e85
Author: Timothy Pearson <kb9vqf@pearsoncomputing.net>
Date:   1330032748 -0600

    Work around Xine crash when displaying still logo image by creating a small movie file to replace it
    This closes Bug 559

diff --git a/kaffeine/images/logo b/kaffeine/images/logo
index f3d53e2..7b76dfe 100644
Binary files a/kaffeine/images/logo and b/kaffeine/images/logo differ
diff --git a/kaffeine/images/logo.still b/kaffeine/images/logo.still
new file mode 100644
index 0000000..f3d53e2
Binary files /dev/null and b/kaffeine/images/logo.still differ
